src/utils/link.ts

```diff
@@ -0,0 +1,12 @@
+export function buildItemLink(projectId: string, languageCodename: string, itemId: string): string {
```

Comment:
Unused exported function, either there is missing handling or item only links somewhere, or this is unnecessary

Reply:
We planned to support links to items (without the need to specify codename of the element), but decided that we won't do this in ETP. We are still planning to add this feature to the SDK in the future, that is why I created this function, but you are right it shouldn't be exported right now.